svg/elements/feComposite.json

```diff
@@ -24,19 +24,19 @@
               "version_added": null
             },
             "opera": {
-              "version_added": false
+              "version_added": "15"
```

Comment:
Chrome has full support from version 1. It got Blink in v28 which corresponds to Opera 15)

Reply:
We do not. Will update.